medcat/cat.py

```diff
@@ -215,6 +224,75 @@ def __call__(self, text: str, do_train: bool = False):
             text = self._get_trimmed_text(str(text))
             return self.pipe(text)
 
+            
+    def save_model(self, vocab_output_file_name="vocab.dat", cdb_output_file_name="cdb.dat", trainer_data_file_name="MedCAT_Export.json", skip_stat_generation=False):
```

Comment:
It could be part of create_model_packs with maybe an optional arg that makes zipping optional ? with the extra addition of having the stats also saved.